packages/dcos-integration-test/extra/test_dcos_log.py

```diff
@@ -1,10 +1,30 @@
 import json
+import logging
 import uuid
 
+from functools import wraps
+
+import pytest
 import requests
 import retrying
 
 
+def dcos_log_enabled(fn):
+    @wraps(fn)
+    def wrapped(dcos_api_session):
+        response = dcos_api_session.get('/dcos-metadata/ui-config.json').json()
+        try:
+            strategy = response['uiConfiguration']['plugins']['mesos']['logging-strategy']
+        except ValueError:
```

Comment:
I think you're looking for `KeyError` for dict key lookup failure. In practice this could fail with other exceptions, since the JSON in the response could contain a list or string where you're expecting a map.

I'd be OK with executing this handler on all exceptions (`except Exception:`) since it always re-raises.